tests/integration/utils.py

```diff
@@ -1,4 +1,38 @@
 from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+DATE_PICKER_DAY_SELECTOR = 'div[data-visible="true"] td.CalendarDay'
+
+
+def choose_calendar_date(dd, comp_id, index=0, day="", outside_month=False):
```

Comment:
@alexcjohnson  I saw your comment about the scope of the calendar API should stay in DCC, but I would prefer to move this in `dash.testing` as it's a very common action we might use elsewhere and requires some tricks to make it work more stable.  another reason I don't like the current utils.py solution is that it requires the extra `__init__.py` be present in the test folders, which makes the test code not clean in a sense that you can run the tests only in the root folder for example.

Reply:
Agreed that adding `__init__.py` is unpleasant, as is the need to explicitly pass in `dash_duo` to the function. But I think most of what happens here is quite specific to DCC and becomes substantially harder to maintain - 2 PRs with coupled versions if the datepicker changes, for example, and if anyone else DOES depend on this API we'd have to find a way to maintain compatibility.

Is there a way to bake this into a DCC-specific version of `dash_duo`, like via `conftest` or something?

> requires some tricks to make it work more stable

If there are *pieces* of this that are transferrable to other scenarios independent of the specific element structure, selectors, etc of DCC, I'll be more than happy to include those in `dash.testing`.

Reply:
yes, we can basically have a dcc fixture like we did in dash-docs, and these `utils` can go to a mixin class like https://github.com/plotly/dash/blob/dev/dash/testing/dash_page.py,  and the fixture will be defined in a `conftest.py` file. this solution would allow the selector to be stayed in dcc, and the abstraction can ship to dash repo.